integration-tests/.storybook/config.js

```diff
@@ -1,4 +1,19 @@
 import { configure, getStorybook } from '@kadira/storybook';
+import inPercy from '@percy-io/in-percy';
+import faker from 'faker';
+import timemachine from 'timemachine';
+
+// This demonstrates how to run code specifically for Percy's rendering environment
+if (inPercy()) {
+  // Seed faker so it generates deterministic fake data
+  faker.seed(123);
+  // Use timemachine to freeze the date to 2015
+  timemachine.config({
+    dateString: 'October 21, 2015 13:12:59'
+  });
+} else {
+  timemachine.reset(); // When we're not in Percy, don't use timemachine
```

Comment:
It is weird that this has side-effects by just being imported, so it's confusing to see it having to be reset instead of just not applied. Maybe comment that those import side-effects exist here and that's why this reset is needed?